### PR TITLE
Hidden API keys and Tokens

### LINKED ISSUE #1: Hide API keys and Tokens
**Is your feature request related to a problem? Please describe.**
As we know API Keys are prone to security vulnerability so we need to hide these API keys and Tokens.

**Describe the solution you'd like**
Refer to this article to add this enhancement [Hide your API and Tokens](https://medium.com/black-tech-diva/hide-your-api-keys-7635e181a06c)

### PR DESCRIPTION
**Pull request by shikharcodes**
Fixes #1 : **I have hidden the API keys and tokens**

Improvements: 

- I have created and added a file config.py and .gitignore 
- I've made few changes in bot.py to successfully hide the API keys and tokens.

### CODE REVIEW

**Comment by DARK-art108** [comment]
@paramsiddharth take a look over the PR

File: bot.py [hunk 1]
```diff
@@ -10,27 +10,19 @@
 from faunadb.objects import Ref
 from faunadb.client import FaunaClient
 import os
-from sys import argv
-
-''' Handling test '''
-if len(argv) > 1:
-    if argv[1] == 'test':
-        print('Imports successful!')
-        exit(0)
```
Comment (paramsiddharth) [inline bot.py]: You removed the test-handling from the code. Please add it again.
Don't forget to add this in the imports:
``` py
from sys import argv
```

**Review by paramsiddharth — COMMENTED** [review]
Minor correction needed in `bot.py` due to information lost due to sync.

**Comment by paramsiddharth** [comment]
Good job. Merged!

**Pull request merged**